openspec/changes/archive/2026-06-19-update-check-silent-not-published/proposal.md

```
@@ -0,0 +1,29 @@
+## Why
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)